root/layout/index.js

```diff
@@ -88,6 +89,30 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user &&
+      $c.user.birth_date &&
```

Comment:
Nitpick, these aren't on the same indentation level as the first condition.

Reply:
I'm never sure what indentation these should have. Eslint seems to like all of them. Changed it, better now?

Reply:
Yeah, that's a bit better. These are a bit annoying so I usually try to move long conditions into a separate variable outside the JSX.